en/flight_modes/README.md

```diff
@@ -409,76 +422,3 @@ Abbreviations:
 
 
 
-<!-- 

```

Comment:
FYI, this was previously commented out. Just removing it now information has been integrated.